### ISSUE TITLE
Checkboxes in the file explorer

### ISSUE THREAD

**Issue by mine-cetinkaya-rundel** [posit-dev/positron#4013]
In the file explorer, it would be nice to have checkboxes next to files and folders for actions like deleting multiple files at once.

<img width="1624" alt="Screenshot 2024-07-14 at 2 49 04 PM" src="https://github.com/user-attachments/assets/e7814759-c893-4019-9107-d8ac6523102c">

**Comment by petetronic** [comment]
Do you mean from a usability perspective? Otherwise, currently you can Cmd+Click to select multiple entries and perform bulk operations.

**Comment by mine-cetinkaya-rundel** [comment]
Yes, I should have clarified that from a usability perspective. (This might be something I'm looking for because I'm coming from RStudio.)
